lib/tools/adb-commands.js

```diff
@@ -835,10 +835,17 @@ methods.push = async function push (localPath, remotePath, opts) {
  *
  * @param {string} remotePath - The source path on the remote device.
  * @param {string} localPath - The destination path to the file on the local file system.
+ * @param {object} opts - Additional options mapping. See
+ *                        https://github.com/appium/node-teen_process,
+ *                        _exec_ method options, for more information about available
+ *                        options.
  */
-methods.pull = async function pull (remotePath, localPath) {
+methods.pull = async function pull (remotePath, localPath, opts) {
```

Comment:
Should here have `opts = {}` as its default value?
(CI failure)

Reply:
Completed 👍